packages/react-instantsearch/src/components/LinkList.js

```diff
@@ -53,6 +53,7 @@ export default class LinkList extends Component {
               item.disabled && 'itemDisabled',
               item.modifier
             )}
+            disabled={item.disabled}
```

Comment:
That way our disabled style mixin is really used. 

![pagination-disabled](https://cloud.githubusercontent.com/assets/6885378/22641618/bef538dc-ec57-11e6-8019-798c7e169399.gif)